### ISSUE TITLE
ZSH Compatibility

### ISSUE THREAD

**Issue by l0010o0001l** [supremebeing7/time_log_robot#37]
`time_log_robot` command executable works on Bash shell, but when in Zsh, this error is rendered:

```
dyld: lazy symbol binding failed: Symbol not found: _rb_str_new_static
  Referenced from: /Users/l0010o0001l/.gem/ruby/2.2.3/gems/json-1.8.3/lib/json/ext/parser.bundle
  Expected in: flat namespace

dyld: Symbol not found: _rb_str_new_static
  Referenced from: /Users/l0010o0001l/.gem/ruby/2.2.3/gems/json-1.8.3/lib/json/ext/parser.bundle
  Expected in: flat namespace

[1]    86607 trace trap  time_log_robot -h
```

![lol](https://cloud.githubusercontent.com/assets/5507015/17374212/625c1dc8-5960-11e6-9f13-7f54aff51082.gif)

**Comment by supremebeing7** [comment]
Sounds like this: https://github.com/AtomLinter/linter-rubocop/issues/84

JSON 1.8.3 seems to have a bug in it that is disrupting the robot. Try uninstalling json 1.8.3 then `gem pristine time_log_robot` and see if that does the trick.

We may need to lock JSON at 1.8.1 until this is fixed.

**Comment by l0010o0001l** [comment]
GEE WILLIKERS that must be it, ha ha ha so ​_SILLY_​.

That works. 

![70366051](https://cloud.githubusercontent.com/assets/5507015/17375398/a187e090-5965-11e6-8f24-0fa32b6041c3.jpg)

**Comment by supremebeing7** [comment]
For anyone else who stumbles on this, another possible solution could be `gem pristine json`.

Lots of discussion on this issue thread on json: https://github.com/flori/json/issues/229

**Comment by supremebeing7** [comment]
Fixed by #39 (hopefully)
